src/main/webapp/i18n/nl/sourceData.json

```diff
@@ -18,8 +18,11 @@
             "sourceDataType": "Source Data Type",
             "sourceDataName": "Source Data Name",
             "processingState": "Processing State",
-            "dataFormat": "Data Format",
-            "frequency": "Frequency",
+            "keySchema": "Key Schema",
+            "valueSchema": "Value Schema",
+            "topic": "Topic",
+            "frequency": "frequentie",
+            "unit": "eenheid",
```

Comment:
Perhaps capitalize the translations for frequency and unit

Reply:
Sure.